website/client/src/components/groups/group.vue

```diff
@@ -256,10 +256,13 @@
     background-color: $gray-600;
     padding-bottom: 2em;
 
+    .below-header-sections {
+        padding: 1em 24px 1em 24px;
```

Comment:
can we use ems everywhere here? So that would be `1em 1.5em 1em 1.5em´

Reply:
actually instead of using a new class this can be achieved by applying `px-4 py-3`

Reply:
same thing in tavern.vue